src/plans/conjugate_residual_plan.jl

```diff
@@ -0,0 +1,335 @@
+#
+#
+# Objective.
+@doc raw"""
+    SymmetricLinearSystemObjective{E<:AbstractEvaluationType,TA,T} <: AbstractManifoldObjective{E}
+
+Model the objective
+
+```math
+f(X) = \frac{1}{2} \lVert \mathcal A[X] + b \rVert_p^2,\qquad X ∈ T_p\mathcal M,
+```
+
+defined on the tangent space ``T_p\mathcal M`` at ``p`` on the manifold ``\mathcal M``.
+
+In other words this is an objective to solve ``\mathcal A(p)[X] = -b(p)``
+for some linear symmetric operator and a vector function.
+Note the minus on the right hand side, which makes this objective especially taylored
```

Comment:
```suggestion
Note the minus on the right hand side, which makes this objective especially tailored
```

Reply:
I do Taylor expansions too much, there you ta_y_lor a function ;)